New package: Fugl v0.4.5

JuliaRegistrator opened the pull request:
- Registering package: Fugl
- Repository: https://github.com/ErikBuer/Fugl.jl
- Created by: @ErikBuer
- Version: v0.4.5
- Commit: f7665a56787c129832f476cff75af6946ea56f7a
- Reviewed by: @ErikBuer
- Reference: https://github.com/ErikBuer/Fugl.jl/issues/67#issuecomment-3225422219
- Description: A functional GUI library written in Julia using OpenGL.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835984ff021207a0a75c88f0a2c9e58585d8bdb74803e4d580bd86dd0ba25e1fd3ee8fb402b5d3f535a5bce17a723705b83c010c8f883c2926db1cdf9dce5aa5b306b68ff3f49a921518c4560dfbdbac7e03541093d6f53f6bf710bd03492cd48ff1ecf80b276738e8cc95f2755f4e60ee2e6 -->

goerz commented:
> Name is not at least 5 characters long

Seems to me the name `Fugl` is okay, and this registration should be merged manually. You will have to fix the other issues, though, and then find a full registry maintainer on Slack to ask for a manual merge, at the end of the three-day waiting period.

ErikBuer commented:
Hi [goerz](https://github.com/goerz).

Thanks for the quick reply.
I will do so! :)

JuliaRegistrator committed:
New package: Fugl v0.4.5

UUID: a78eb1c3-f9be-4b1a-a708-e7053b7f16d7
Repo: https://github.com/ErikBuer/Fugl.jl.git
Tree: 761666df3e09a5f1e1c492f9d9043acf5166ff74

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
